src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,24 +1,102 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 /**
  * <p>Реалізувати свою HashMap, а саме методи `put(K key, V value)`, `getValue()` та `getSize()`.
  * Дотриматися основних вимог щодо реалізації мапи (initial capacity, load factor, resize...)
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int initialCapacity = 16;
```

Comment:
is it constant? - https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names

Reply:
I thought it is, like in array list, 
but don`t wonder why?